### PR TITLE
chore(blog): Removing UDP info from the ILP blog

### PR DESCRIPTION
**Pull request by amyshwang**

### CODE REVIEW

File: blog/2020-07-22-influxdb-lp-on-tcp.mdx [hunk 1]
```diff
@@ -11,8 +11,10 @@ keywords:
   - timeseries
   - questdb
   - influxdb
-tags: [tutorial, questdb, influxdb]
+  - ilp
+tags: [tutorial, questdb, influxdb, lip]
```
Comment (tris0laris) [inline blog/2020-07-22-influxdb-lp-on-tcp.mdx:15]: typo, with lip instead of ilp in the tag

File: blog/2020-07-22-influxdb-lp-on-tcp.mdx [hunk 2]
```diff
@@ -28,21 +30,10 @@ import Banner from "@theme/Banner"
...
-
-We've had a UDP version of the InfluxDB Line Protocol (ILP) reader in QuestDB
-for quite some time, but we've had customers ask for a TCP version of it, so we
-delivered!
-
-Using and configuring it are relatively simple, so this tutorial will walk you
-through the basics of setting it up and using it. For an added bonus, we'll see
-how to migrate from using InfluxDB to using QuestDB with a single-line
-configuration change.
+This tutorial will walk you through the basics of setting it up and using the
```
Comment (tris0laris) [inline blog/2020-07-22-influxdb-lp-on-tcp.mdx:33]: Setting what up? We have not introduced ILP yet, this is the first sentence

File: blog/2020-07-22-influxdb-lp-on-tcp.mdx [hunk 3]
```diff
@@ -153,31 +145,59 @@ This is what that table looks like in the QuestDB Web Console:
...
+                    'dev_name': 'BME280',
+                    'temp_c': 23.18,
+                    'altitude': 93.10,
+                    'humidity': 52.16,
+                    'pressure': 1002.12})
+            sender.flush()
+
+    except IngressError as e:
+        sys.stderr.write(f'Got error: {e}\n')
+
```
Comment (tris0laris) [inline blog/2020-07-22-influxdb-lp-on-tcp.mdx:181]: thanks. Have you tested it to make sure that yields the expected results? im sure you did, but making sure

File: blog/2020-07-22-influxdb-lp-on-tcp.mdx [hunk 4]
```diff
@@ -153,31 +145,59 @@ This is what that table looks like in the QuestDB Web Console:
...
-in a `buffer` I can call `espClient.printf(buffer);` and that line of data will
-be written to QuestDB.
+`sender.flush()` flushes the internal buffer and we recommend flushing
+periodically, for example every few seconds.
 
 ## Can I do batch writes?
 
-Of course, you can! Put each line of ILP on a separate 'line', separated by a
+Of course, you can! Put each line of ILP on a separate line, separated by a
 newline `\n`, and then when you have all your batch built up, write the whole
```
Comment (tris0laris) [inline blog/2020-07-22-influxdb-lp-on-tcp.mdx:195]: newline: is it newline or new line? newline accepted term? i dont know btw

File: blog/2020-07-22-influxdb-lp-on-tcp.mdx (hunk 1 above)
Comment (tris0laris) [inline blog/2020-07-22-influxdb-lp-on-tcp.mdx:15]: could we also add telegraf in the tags?

File: blog/2020-07-22-influxdb-lp-on-tcp.mdx (hunk 3 above)
Comment (amyshwang) [reply blog/2020-07-22-influxdb-lp-on-tcp.mdx:183]: Hmm I did not test the error handling part as I don't know how to. Nevertheless, this is copied from our example - https://questdb.io/docs/develop/insert-data/#examples. I will ask for help with this part of the code.

File: blog/2020-07-22-influxdb-lp-on-tcp.mdx (hunk 4 above)
Comment (amyshwang) [reply blog/2020-07-22-influxdb-lp-on-tcp.mdx:197]: It refers to the character `/n` used to indicate the end of a line so I'd keep it as it is!

**Pull request closed without merging**